shardingsphere-distsql/shardingsphere-distsql-parser/src/main/antlr4/imports/RQLStatement.g4

```diff
@@ -34,6 +34,10 @@ showAuthorityRule
     : SHOW AUTHORITY RULE
     ;
     
+showTransactionRule
+    : SHOW TRANSACTION RULE (FROM schemaName)?
```

Comment:
`TRANSACTION RULE` is  a global rule, dose not need `from schema`

Reply:
> `TRANSACTION RULE` is a global rule, dose not need `from schema`

OK, I will adjust.